yt_dlp/utils.py

```diff
@@ -2063,7 +2063,8 @@ def sanitize_open(filename, open_mode):
                 import msvcrt
                 msvcrt.setmode(sys.stdout.fileno(), os.O_BINARY)
             return (sys.stdout.buffer if hasattr(sys.stdout, 'buffer') else sys.stdout, filename)
-        stream = open(encodeFilename(filename), open_mode)
+        stream = locked_file(encodeFilename(filename), open_mode, block=False)
+        stream = stream.__enter__()
```

Comment:
```suggestion
        stream = locked_file(filename, open_mode, block=False).open()
```
Perhaps this is better syntax (similar to `.close`)

(`encodeFilename` is a no-op)

Reply:
Change made, as a future TODO `class locked_file()` is a context manager, and as you mentioned https://github.com/yt-dlp/yt-dlp/pull/1066#discussion_r714647420 it's not being used as one.